tests/contrib/dbapi/test_dbapi.py

```diff
@@ -522,15 +522,16 @@ def test_callproc_can_handle_arbitrary_args(self):
 
     @TracerTestCase.run_in_subprocess(
         env_overrides=dict(
-            DD_DBM_PROPAGATION_MODE="full",
+            DD_DBM_PROPAGATION_MODE="service",
```

Comment:
is this a separate change? Or does it relate to the decoupling?

Reply:
It's a separate change that I forgot to revert. Nice catch